components/countdown/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+
+import Box from '@components/box';
+
+const getTargetDate = (): Date => {
+	const now = new Date();
+	const sundayNumber = 0;
+	const timezone = 'America/New_York';
+	const today = new Date(now);
+	today.setMilliseconds(0);
+	today.setSeconds(0);
+	today.setMinutes(0);
+	today.setHours(0);
+
+	const nextSunday = new Date(today);
+
+	do {
+		nextSunday.setDate(nextSunday.getDate() + 1);
+	} while (nextSunday.getDay() !== sundayNumber);
+
+	return new Date(nextSunday.toLocaleString('en-US', { timeZone: timezone }));
+};
+
+const getValues = (countDown: number): [string, string, string, string] => {
+	const days = Math.floor(countDown / (1000 * 60 * 60 * 24)).toLocaleString('en-US');
+
+	const hours = Math.floor((countDown % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)).toLocaleString('en-US', {
+		minimumIntegerDigits: 2,
+		useGrouping: false,
+	});
+
+	const minutes = Math.floor((countDown % (1000 * 60 * 60)) / (1000 * 60)).toLocaleString('en-US', {
+		minimumIntegerDigits: 2,
+		useGrouping: false,
+	});
+
+	const seconds = Math.floor((countDown % (1000 * 60)) / 1000).toLocaleString('en-US', {
+		minimumIntegerDigits: 2,
+		useGrouping: false,
```

Comment:
These static numbers should be extracted into their own variables and perhaps hoisted out of this function if they're being used. e.g:

```ts
const oneSecond = 1000
const oneMinute = oneSecond * 60
const oneHour = oneMinute * 60
const oneDay = oneHour * 24
```

Reply:
👍 https://github.com/ardriveapp/inferno-leaderboard/pull/2/commits/805e1f9541a9050fc53f84aa5c6f2d61d083ef85